package.json

```diff
@@ -6,7 +6,6 @@
     "@babel/preset-env": "^7.17.10",
```

Comment:
- [ ] [OPTIONAL] Good job 🙌🏼💥. There are some changes you need to fix in the MIT.md file. Let's change the year to 2022, put your name and change app time 👍